src/main/java/frc/robot/end_effector/EndEffector.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.end_effector;
+
+import org.littletonrobotics.junction.Logger;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.end_effector.EndEffectorIO.EndEffectorIOInputs;
+
+public class EndEffector extends SubsystemBase {
+    
+    private final EndEffectorIO io;
+    private final EndEffectorIOInputs inputs = new EndEffectorIOInputs();
```

Comment:
You should be used the generated auto-log class instead of `EndEffectorIOInputs`. https://docs.advantagekit.org/data-flow/recording-inputs/annotation-logging/